### ISSUE TITLE
Pitch glue in Bowl crashes client on repair attempt

### ISSUE THREAD

**Issue by Agent00Ming** [anegostudios/VintageStory-Issues#3588]
### Game Version

1.19.4-rc1, 1.19.4-rc2

### Platform

Linux

### Modded

Vanilla

### SP/MP

Singleplayer

### Description

'Pitch glue' inside of 'Fired Crock' and 'Residue-covered pot' work as expected when repairing clutter blocks inside ruins but using a 'Bowl with Meal' containing 'Pitch glue' to repair a clutter block results in a crash. (Only in survival, it does not crash in creative.)

### How to reproduce

Cook up at least one portion/serving of 'Pitch glue' in a cooking pot. Transfer said portion to a bowl and attempt to repair a clutter block in a ruin with it while in survival. Experience a crash.

### Logs

```Running on 64 bit Linux (Arch Linux) [Kernel 6.7.4.1] with 31824 MB RAM
Game Version: v1.19.4-rc.1 (Unstable)
2/15/2024 4:37:53PM: Critical error occurred
Loaded Mods: game@1.19.4-rc.1, creative@1.19.4-rc.1, survival@1.19.4-rc.1
System.NullReferenceException: Object reference not set to an instance of an object.
   at Vintagestory.API.Common.AssetLocation.ResolveToDomainAndPath(String domainAndPath, String& domain, String& path) in VintagestoryApi\Common\Assets\AssetLocation.cs:line 170
   at Vintagestory.GameContent.BlockBehaviorReparable.OnBlockInteractStart(IWorldAccessor world, IPlayer byPlayer, BlockSelection blockSel, EnumHandling& handling) in VSSurvivalMod\BlockBehavior\BehaviorReparable.cs:line 153
   at Vintagestory.API.Common.Block.OnBlockInteractStart(IWorldAccessor world, IPlayer byPlayer, BlockSelection blockSel) in VintagestoryApi\Common\Collectible\Block\Block.cs:line 1346
   at Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.TryBeginUseBlock(Block selectedBlock, BlockSelection blockSelection) in VintagestoryLib\Client\Systems\Player\MouseInWorldInteractions.cs:line 864
   at Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.HandleMouseInteractionsBlockSelected(Single dt) in VintagestoryLib\Client\Systems\Player\MouseInWorldInteractions.cs:line 444
   at Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.OnFinalizeFrame(Single dt) in VintagestoryLib\Client\Systems\Player\MouseInWorldInteractions.cs:line 89
   at Vintagestory.Client.NoObf.ClientEventManager.TriggerRenderStage(EnumRenderStage stage, Single dt) in VintagestoryLib\Client\Util\ClientEventManager.cs:line 186
   at Vintagestory.Client.NoObf.ClientMain.TriggerRenderStage(EnumRenderStage stage, Single dt) in VintagestoryLib\Client\ClientMain.cs:line 807
   at Vintagestory.Client.NoObf.ClientMain.RenderToDefaultFramebuffer(Single dt) in VintagestoryLib\Client\ClientMain.cs:line 1011
   at Vintagestory.Client.ScreenManager.Render(Single dt) in VintagestoryLib\Client\ScreenManager.cs:line 676
   at Vintagestory.Client.ScreenManager.OnNewFrame(Single dt) in VintagestoryLib\Client\ScreenManager.cs:line 651
   at Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(FrameEventArgs e) in VintagestoryLib\Client\ClientPlatform\GameWindow.cs:line 88
   at OpenTK.Windowing.Desktop.GameWindow.Run()
   at Vintagestory.Client.ClientProgram.Start(ClientProgramArgs args, String[] rawArgs) in VintagestoryLib\Client\ClientProgram.cs:line 324
   at Vintagestory.Client.ClientProgram.<>c__DisplayClass9_0.<.ctor>b__1() in VintagestoryLib\Client\ClientProgram.cs:line 128
   at Vintagestory.ClientNative.CrashReporter.Start(ThreadStart start) in VintagestoryLib\Client\ClientPlatform\ClientNative\CrashReporter.cs:line 93
```

**Comment by Agent00Ming** [comment]
Issue can be reproduced in 1.19.4-rc2, log:
```Running on 64 bit Linux (Arch Linux) [Kernel 6.7.5.1] with 31824 MB RAM
Game Version: v1.19.4-rc.2 (Unstable)
2/22/2024 7:17:00PM: Critical error occurred
Loaded Mods: game@1.19.4-rc.2, creative@1.19.4-rc.2, survival@1.19.4-rc.2
System.NullReferenceException: Object reference not set to an instance of an object.
   at Vintagestory.API.Common.AssetLocation.ResolveToDomainAndPath(String domainAndPath, String& domain, String& path) in VintagestoryApi\Common\Assets\AssetLocation.cs:line 170
   at Vintagestory.GameContent.BlockBehaviorReparable.OnBlockInteractStart(IWorldAccessor world, IPlayer byPlayer, BlockSelection blockSel, EnumHandling& handling) in VSSurvivalMod\BlockBehavior\BehaviorReparable.cs:line 153
   at Vintagestory.API.Common.Block.OnBlockInteractStart(IWorldAccessor world, IPlayer byPlayer, BlockSelection blockSel) in VintagestoryApi\Common\Collectible\Block\Block.cs:line 1361
   at Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.TryBeginUseBlock(Block selectedBlock, BlockSelection blockSelection) in VintagestoryLib\Client\Systems\Player\MouseInWorldInteractions.cs:line 885
   at Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.HandleMouseInteractionsBlockSelected(Single dt) in VintagestoryLib\Client\Systems\Player\MouseInWorldInteractions.cs:line 444
   at Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.OnFinalizeFrame(Single dt) in VintagestoryLib\Client\Systems\Player\MouseInWorldInteractions.cs:line 89
   at Vintagestory.Client.NoObf.ClientEventManager.TriggerRenderStage(EnumRenderStage stage, Single dt) in VintagestoryLib\Client\Util\ClientEventManager.cs:line 186
   at Vintagestory.Client.NoObf.ClientMain.TriggerRenderStage(EnumRenderStage stage, Single dt) in VintagestoryLib\Client\ClientMain.cs:line 809
   at Vintagestory.Client.NoObf.ClientMain.RenderToDefaultFramebuffer(Single dt) in VintagestoryLib\Client\ClientMain.cs:line 1013
   at Vintagestory.Client.ScreenManager.Render(Single dt) in VintagestoryLib\Client\ScreenManager.cs:line 676
   at Vintagestory.Client.ScreenManager.OnNewFrame(Single dt) in VintagestoryLib\Client\ScreenManager.cs:line 651
   at Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(FrameEventArgs e) in VintagestoryLib\Client\ClientPlatform\GameWindow.cs:line 88
   at OpenTK.Windowing.Desktop.GameWindow.Run()
   at Vintagestory.Client.ClientProgram.Start(ClientProgramArgs args, String[] rawArgs) in VintagestoryLib\Client\ClientProgram.cs:line 324
   at Vintagestory.Client.ClientProgram.<>c__DisplayClass9_0.<.ctor>b__1() in VintagestoryLib\Client\ClientProgram.cs:line 128
   at Vintagestory.ClientNative.CrashReporter.Start(ThreadStart start) in VintagestoryLib\Client\ClientPlatform\ClientNative\CrashReporter.cs:line 93
```

**Comment by radfast** [comment]
Thanks for reporting, this will be fixed in 1.19.4-stable when it comes!
